modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-creating-ostree-iso_{context}"]
+= Creating the {op-system-ostree-first} image
+
+Use the following procedure to create the ISO. The {op-system-ostree} Installer image pulls the commit from the running container and creates an installable boot ISO with a Kickstart file configured to use the embedded OSTree commit.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
+* You have the `podman` tool.
+
+.Procedure
+
+. Start an `ostree` container image build by running the following command:
++
+[source,terminal]
+----
+BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/9/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
```

Comment:
Is there an extra step not documented ? When I run this command, I receive the following response

_ERROR: UnknownBlueprint: Unknown blueprint name: minimal-microshift_

Reply:
@ggiguash I think this speaks to what Doug noticed. Do we need to move some steps around?

Reply:
@dhensel-rh I think we have the step you need in the next module, adding service to blueprint:

https://61003--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree.html#adding-microshift-service_microshift-embed-in-rpm-ostree

So maybe we need a separate module for creating the blueprint that is called before the image creation of the ostree iso. @ggiguash What do you think?

Reply:
Yes, this is exactly the error I saw. I think if you swap the order of the existing modules it will fix the problem, right?

Reply:
Will it? Can we do all the steps of adding the service the blueprint first? Or do we need to 
1) create the blueprint, 
2) create the ostree iso, then 
3) add the service to the blueprint?

Reply:
Yes, swap the order (and possibly combine the two procedures into one procedure). I tried this on a fresh machine. The minimal-microshift blueprint needs to exist first, since it is used in Creating the Red Hat Enterprise Linux (RHEL) for Edge image, _Procedure,_ Step 1

Example Image Builder blueprint with MicroShift
Creating the Red Hat Enterprise Linux (RHEL) for Edge image

Reply:
@dhensel-rh and @ggiguash I swapped the order of the modules. Please check the steps.

Reply:
@dhensel-rh I'll resolve this comment because you are retesting with the command changes from @dhellmann.

Reply:
@ShaunaDiaz I'm ok with resolving